From 38ccd92d91e2f071efcf54b078f9a8e9455e4861 Mon Sep 17 00:00:00 2001
From: Kevin Mark <kmark937@gmail.com>
Date: Tue, 2 Jul 2019 14:22:23 -0400
Subject: [PATCH 11/49] If we're using a cvs snapshot which updates the source
 files, and perl isn't installed yet, then we can't regen the docs.  Not a big
 deal, so just whine a little and continue on our merry way.

http://bugs.gentoo.org/60132
---
 manual/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/manual/Makefile b/manual/Makefile
index c5866eb9de..39e7a9bf6d 100644
--- a/manual/Makefile
+++ b/manual/Makefile
@@ -104,9 +104,14 @@ dir-add.texi: xtract-typefun.awk $(texis)
 libm-err.texi: stamp-libm-err
 stamp-libm-err: libm-err-tab.pl $(wildcard $(foreach dir,$(sysdirs),\
 						     $(dir)/libm-test-ulps))
+ifneq ($(PERL),no)
 	pwd=`pwd`; \
 	$(PERL) $< $$pwd/.. > libm-err-tmp
 	$(move-if-change) libm-err-tmp libm-err.texi
+else
+	echo "Unable to rebuild math docs, no perl installed"
+	touch libm-err.texi
+endif
 	touch $@
 
 # Generate Texinfo files from the C source for the example programs.
-- 
2.22.0

